### PR TITLE
OpenPathSampling 1.6.1

### PR DESCRIPTION
**Pull request by dwhswenson**
OpenPathSampling 1.6.1 is a minor update, primarily focused on allowing OPS to run/be tested on Python 3.12. In practice, that means removing dependency on tools that prevented Python 3.12 support. In particular, this removes `nose` from our testing requirements, and deprecates our integration with PyEMMA, which is no longer maintained.

**DEPRECATION:** As of OPS 1.6.1, we no longer recommend the use of `PyEMMAFeaturizerCV`. PyEMMA is no longer maintained. Please do not create collective variables based on PyEMMA.

## Miscellaneous improvements

* Test against Python 3.12; deprecate PyEMMA ([#1145](https://github.com/openpathsampling/openpathsampling/pull/1145) #Deprecations)
* Cut off `nose` ([#1141](https://github.com/openpathsampling/openpathsampling/pull/1141) #upstream_changed)
* Fix a few more setup -> setup_method tests ([#1142](https://github.com/openpathsampling/openpathsampling/pull/1142) #upstream_changed)

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -55,6 +55,3 @@ test =
     nbval>=0.10.0
 simstore =
     # simstore requirements have been moved to main
-
-[bdist_wheel]
-universal = 1
```
Comment (sroet) [inline setup.cfg:61]: Why was this removed?

**Review by sroet — APPROVED** [review]
(I only checked the last 4 commits: 878620a - 3584801)
One question, not a blocker. LGTM otherwise

File: setup.cfg (hunk 1 above)
Comment (dwhswenson) [reply setup.cfg:61]: Turns out I'd misremembered what `bdist_wheel.universal` does.  It isn't the equivalent of noarch; it is noarch + compatible with Python 2 and 3. We're no longer compatible with Python 2. With `universal = 1`, wheels claim to be Python 2 compatible, even if `python_requires` is >3 (e.g., https://pypi.org/project/openpathsampling/1.6.0/#files)

I probably knew that when I first added that to a project, but kinda forgot.
